pkg/algorithm/nodeselect/v1alpha1/config.go

```diff
@@ -35,10 +35,10 @@ const (
 
 // DefaultDiskCount is a map containing the default disk count of various raid types.
 var DefaultDiskCount = map[string]int{
-	string(apis.PoolTypeMirroredCPV): int(apis.MirroredDiskCountCPV),
-	string(apis.PoolTypeStripedCPV):  int(apis.StripedDiskCountCPV),
-	string(apis.PoolTypeRaidzCPV):    int(apis.RaidzDiskCountCPV),
-	string(apis.PoolTypeRaidz2CPV):   int(apis.Raidz2DiskCountCPV),
+	string(apis.PoolTypeMirroredCPV): int(apis.MirroredBlockDeviceCountCPV),
```

Comment:
Please check if this can be taken up in this PR or perhaps next PR.
This is about namings:
`PoolTypeMirror`, `DefaultBlockDeviceCountMirror` & so on

Reply:
Naming changes will be done in next PR's If it is required
cc: @sonasingh46